runtime/src/runtime_api.rs

```diff
@@ -459,16 +459,54 @@ mod tests {
             .fold(0, |acc, x| acc.checked_add(*x).unwrap());
     }
 
-    #[ignore]
     #[test]
     fn call_size() {
-        let size = core::mem::size_of::<Call>();
+        // Target limit - substrate frame pallets only Call size = 208
+        const LIMIT: usize = 120;
```

Comment:
Why is `120` a target limit, given that so many of our pallets are already above it?
Shouldn't we use, for example, a number that we know is safe given our current wasm heap size?

Reply:
target limit was probably a bad choice of name. The intent was to just try to have as many pallets below the "clippy" recommended max of ~ 200 I think.

Reply:
Do you have a figure I can plug in, based on your past "tests"?

Reply:
I was able to overflow a 512 MiB wasm memory limit with `880,000` calls of our current call size (~`360 bytes`)
`880,000 * 360 / 1024 / 1024 = 302 MiB`
So I think we should aim for `max_possible_batched_calls * call_size_bytes  <= 1 / 2 * wasm_heap_limit_bytes`.
Since our current `wasm_heap_limit` is `1 GiB` and `max_possible_batched_calls` is ~`1,250,000` (due to block size limit) we get:
```
1,250,000 / 1024 / 1024 * x <= 512
x = ~429
```

So I think we shouldn't go much higher than `400` to be absolutely safe.

Reply:
Brilliant, I'll update the test.